packages/client/src/runtime/utils/validatePrismaClientOptions.ts

```diff
@@ -61,9 +61,14 @@ It should have this form: { url: "CONNECTION_STRING" }`,
     }
   },
   adapter: (adapter, config) => {
-    if (adapter === undefined) {
+    if (adapter === null) {
       return
     }
+    if (adapter === undefined) {
+      throw new PrismaClientConstructorValidationError(
+        `"adapter" property must not be undefined, use null to conditionally disable driver adapters.`,
```

Comment:
Additionally, we could also mention: Or remove the `driverAdapters` preview feature flag?

Reply:
(though that would not be an option once it's GA)

Reply:
Without the feature flag though it would be an unknown option.

Reply:
at least on the type level